### DISCUSSION TITLE
change the physics engine?

### DISCUSSION THREAD

**Discussion by vitalspace** [enable3d/enable3d#275] [Ideas]
Are there plans to change the "bullet" physics engine to "rapier3d"? The other day I came across this comparison.

https://youtu.be/SOFUgOkuHW8

**Comment by yandeu** [comment]
It's a strange comparison. Bullet is a very very good and mature engine used by many games.
The importance is not how many objects can collide with each other but how these objects behave. Bullet is very realistic.

So the answer is no.

**Reply by vitalspace** [reply]
Okay

**Reply by vitalspace** [reply]
well maybe it's not the plan to change the physics engine, but is it possible to use the rapier physics in enable3D?

**Reply by yandeu** [reply]
enable3d is just a wrapper around three.js. If rapier works with three.js it will also work with enable3d.
